client/my-sites/hosting/main.js

```diff
@@ -76,10 +76,9 @@ class Hosting extends Component {
 
 		const getUpgradeBanner = () => (
 			<UpsellNudge
-				title={ translate( 'Upgrade to the Business plan to access all hosting features' ) }
+				title={ translate( 'Upgrade your plan to access all hosting features' ) }
 				event="calypso_hosting_configuration_upgrade_click"
-				href={ `/checkout/${ siteId }/business` }
-				plan={ PLAN_BUSINESS }
+				href={ `/plans/${ siteSlug }/` }
```

Comment:
I think we should leverage the `feature` URL parameter to ensure that the correct feature and plan(s) are highlighted in the plans page.
Also, I think the URL used in the left nav does not include a trailing `/`.
```suggestion
				// import { FEATURE_SFTP_DATABASE } from '@automattic/calypso-products';
				href={ `/plans/${ siteSlug }?feature=${ encodeURIComponent( FEATURE_SFTP_DATABASE ) }` }
```

Reply:
I updated the import to use the broader feature set and added it as a query param. I thought there was supposed to be a visible highlight on the Plans page based on that query param but I'm not seeing one. Maybe I'm misremembering!

Reply:
@daledupreez Would it make sense to add the `&plan=${ encodeURIComponent( PLAN_BUSINESS ) }` query so we're only showing Business + eComm (plans that have the feature) when users land on the Plans page?

<img width="1230" alt="Screen Shot 2023-01-11 at 9 45 41 AM" src="https://user-images.githubusercontent.com/2124984/211835819-7789e64b-0328-4745-9a7f-aa945cf52617.png">

Reply:
This feels like it might cause us problems if we go down this path, especially if we're not going to show the Business plan to users within the free eCommerce trial.